docs/marketplace/user/features/202106.0/merchant-products.md

```diff
@@ -0,0 +1,78 @@
+
+1. The product status is *Active*.
+2. The merchant who owns the product is [*Active*](https://spryker-docs.herokuapp.com/docs/marketplace/user/back-office-user-guides/202106.0/marketplace/merchants/managing-merchants.html#activating-and-deactivating-merchants).
+3. The product visibility state is `Online`.
+4. The product is defined for the current store.
+5. The product has stock or is always in stock.
+6. The current day is within the range of the product validity dates.
+
+### Merchant product on the product details page
+
+Merchant product appears on top of the the *Sold by* list together with the product offers from other merchants. For a Buyer, it doesn't matter whether they are buying a product offer or a merchant product, however in the system, different entities are defined.
+
+Product price on top of the product details page is taken from the merchant product or the product offer. It depends on the option selected in the *Sold by* box.
+
+![Merchant product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/merchant-product-on-pdp.png)
+
+The merchant product is also displayed with the *Sold By* field defining the merchant on the following pages:
+
+- Cart page
+- Wishlist
+- Shipment page of the Checkout
+- Summary page of the Checkout
+- Order Details of the customer account
+
+![Merchant product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/add-merchant-product-to-wl-and-from-wh-to-cart.gif)
+
+
+### Searching and filtering merchant products
+When the merchant name is entered in the catalog search, not only the offers but also the products belonging to this merchant are displayed. By selecting a merchant name in the filter, products from this merchant are also displayed.
+
+![Search for merchant products](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/search-for-products-by-name-and-sku.gif)
+
+
+## Merchant products in the Back Office
+Before new merchant products can be seen by the Buyers in the Storefront, they must be activated <!---LINK--> in the Back Office.
+
+A Marketplace Administrator can filter the products belonging to certain merchants in the Back Office.
+
+![merchants-switcher-on-products](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/filter-merchant-productsby-merchant-back-office.gif)
+
+Also, Marketplace administrators can edit products, if needed, and create products when acting as the main merchant<!---LINK-->.
+
+## Merchant products in the Merchant Portal
+Merchants create and manage their products <!---LINK--> in the Merchant Portal. They can define their own prices, stock, attributes etc. for their products. See Managing merchant products <!---LINK--> for detailed instructions on how to do that.
+
+## Merchant products and API
```

Comment:
This section can be moved to the Marketplace Products feature walkthrough for devs.